src/Sylius/Bundle/PayumBundle/Model/GatewayConfigInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\PayumBundle\Model;
+
+use Payum\Core\Model\GatewayConfigInterface as BaseGatewayConfigInterface;
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+interface GatewayConfigInterface extends BaseGatewayConfigInterface, ResourceInterface
```

Comment:
I suppose it should be now implemented by `Sylius\Bundle\PayumBundle\Model\GatewayConfig`?

Reply:
Done 👍